Sage/carbon
# docs(animated menu button): remove classic theme from storybook

fixes FE-2903

### Proposed behaviour

Removal of classic theme story from storybook

### Current behaviour

Classic theme story is present for Animated menu Button

### Checklist

<del>- [ ] Screenshots are included in the PR </del>
<del>- [ ] Carbon implementation and Design System documentation are congruent</del>
<del>- [ ] All themes are supported</del>
<del>- [ ] Unit tests added or updated</del>
<del>- [ ] Cypress automation tests added or updated</del>
<del>- [ ] Typescript `d.ts` file added or updated</del>

### Additional context
N/A

### Testing instructions
- Pull down branch
- Open up storybook using `npm start`
- Check that Animated Menu Button component doesn't have a classic story
